src/game/objects/general/scion1.c

```diff
@@ -0,0 +1,85 @@
+#include "game/objects/general/scion1.h"
+
+#include "game/input.h"
+#include "game/inventory.h"
+#include "game/items.h"
+#include "game/lara.h"
+#include "game/objects/common.h"
+#include "game/overlay.h"
+#include "global/vars.h"
+
+#define EXTRA_ANIM_PEDESTAL_SCION 0
+#define LF_PICKUPSCION 44
+
+static XYZ_32 m_Scion1_Position = { 0, 640, -310 };
+
+static const OBJECT_BOUNDS m_Scion1_Bounds = {
+    .shift = {
+        .min = { .x = -256, .y = +640 - 100, .z = -350, },
+        .max = { .x = +256, .y = +640 + 100, .z = -200, },
+    },
+    .rot = {
+        .min = { .x = -10 * PHD_DEGREE, .y = 0, .z = 0, },
+        .max = { .x = +10 * PHD_DEGREE, .y = 0, .z = 0, },
+    },
+};
+
+static const OBJECT_BOUNDS *Scion1_Bounds(void);
+
+static const OBJECT_BOUNDS *Scion1_Bounds(void)
+{
+    return &m_Scion1_Bounds;
+}
+
+void Scion1_Setup(OBJECT_INFO *obj)
```

Comment:
What do you think about having some sort of comment in each Scion file describing which one it is? Example - Qualopec pedestal Scion, Pierre dropped Scion, etc? Just the number has been confusing in the past. @lahm86 what do you think also?

Reply:
I'm all for it, I just don't know what each of them does :D

Reply:
That would be great 😌 

- `O_SCION_ITEM` - ToQ and Sanctuary, triggers `O_LARA_EXTRA` pedestal pickup anim
- `O_SCION_ITEM2` - normal pickup
- `O_SCION_ITEM3` - the Great Pyramid shootable variety
- `O_SCION_ITEM4` - Atlantis, triggers `O_LARA_EXTRA` reach anim

Reply:
Done